tests/epyccel/test_epyccel_variable_annotations.py

```diff
@@ -122,8 +122,7 @@ def final_annotation():
 
 def test_homogeneous_tuple_annotation(language):
     def homogeneous_tuple_annotation():
-        # Not valid in Python 3.8
-        a : tuple[int, ...] #pylint: disable=unsubscriptable-object
+        a : tuple[int, ...]
         a = (1,2,3)
```

Comment:
Does the inline annotation work, too?

Reply:
Yes I think so. Odd that it is only tested for inhomogeneous tuples